jwql/tests/test_database_interface.py

```diff
@@ -48,11 +50,14 @@ def test_all_tables_exist():
             pass  # Not all attributes of database_interface are table ORMs
 
     # Get list of tables that are actually in the database
-    existing_tables = di.engine.table_names()
+    existing_tables = inspect(di.engine).get_table_names()
 
     # Ensure that the ORMs defined in database_interface actually exist
     # as tables in the database
     for table in table_orms:
+        # TODO: table doesn't exist on dev server
+        if table in ['nirspec_ta_stats']:
```

Comment:
In general we should probably make sure that all our databases contain all the tables, but in this case, I'm not convinced that nirspec_ta_stats is actually used anywhere. I see the MSATA monitor connecting to it, but I don't see any data actually being put into it. I've asked Maria to confirm. 
So it seems like we could add the table to the dev server so that we don't need any conditional clause here, and then in a separate PR we can remove the table completely if Maria says we don't need it? What do you think?

Reply:
I agree, I think for this case we can push this through and when Maria gets back to us we can submit the PR and update this test as part of it.

Reply:
Maria reports that the nirspec_ta_stats table is not used at all. So I think we can add the table to the dev server and delete these two lines. Then we won't need to worry about this later, when we remove the table completely.

Reply:
Thanks @bhilbert4 should we just delete the table as well as remove the lines?

Reply:
I've removed the lines from the test and confirmed that the test passes when run against the production database.  It currently fails when run against dev; I can check again if/when we add the table there.